---
layout: group
author: smb2023
day: "Wednesday"
subgroup: "CDEV"
subgroup-long: "CDEV-2"
title: "The role of the microenvironment in controlling cell phenotypic decisions across scales"
organizers: "Laura F. Strube, Adam L. MacLean"
description: "Cells make fate decisions and phenotypic changes throughout life. Particularly prominent are their roles in development, immune responses, and cancer progression. In each case, the outcome of a decision depends on a complex interplay between signal transduction pathways within cells and feedbacks to and from the surrounding microenvironment. Some cell state transitions might follow “simple” trajectories, e.g. between binary fates, but increasingly, evidence points towards wider spectra of accessible cell states under complex cell- and environment-dependent control. Additionally, transcriptional states are not always sufficient to describe these states: post-transcriptional regulation via epigenetic modifications, microRNAs, etc. also play important roles in determining cell fate. Thus, understanding and predicting how tissue microenvironmental signals impact cell phenotypes demand a systems-level perspective that integrates mathematical modeling with quantitative tools. This minisymposium highlights the state-of-the-art in this domain: coupling quantitative experimental and computational methods to reveal the molecular underpinnings of cell fate decision-making phenomena. The talks comprising this session will describe discoveries in model systems ranging from pro- vs. anti-inflammatory responses in macrophages and the epithelial-to-mesenchymal transition in development, to the multiscale effects of microenvironmental signaling on cell fate decisions in the developing mammary gland. Innovative approaches these works employ include multiscale modeling, genomics analyses, and new methods for machine learning coupled with differential equation modeling. Overall, these talks will reveal new understanding into dynamic cell phenotypes through quantitative modeling across biological scales."
room: "Barbie Tootle Room (#3156)"
code: "MS05"
author1: "Tian Hong"
inst1: "The University of Tennessee, Knoxville (Biochemistry & Cellular and Molecular Biology)"
title1: "Diverse dynamical systems for understanding nongenetic heterogeneity of cells"
additional_authors1: ""
abstract1: "The phenotypic heterogeneity within cell populations, both signal-induced and self-generated, plays crucial roles in development, cancer progression, and drug resistance. However, our fundamental understanding of these phenomena at the dynamical systems level remains limited. Epithelial-mesenchymal transition (EMT) is one example of a cellular process that triggers heterogeneity-driving phenotypic changes. While multiple intermediate/hybrid EMT states have been observed in development and diseases, it is still unclear whether these intermediate states represent transient states for cells en route to M-like cells or stable phenotypes representing ordered attractors between E and M states. Our recent single-cell experiments with human mammary epithelial cells and analysis of published data have shown that both transient states and ordered attractors can explain intermediate EMT states. Additionally, our mathematical models of widespread RNA-decay regulatory networks have demonstrated that slow oscillations with diverging periods can drive self-generated heterogeneity in cell populations, achieving phenotypic diversity and multimodal gene expression patterns more robustly than commonly conceptualized multistability systems. This theoretical framework provides insight into the observations of heterogeneity in progenitor cells and cancer cells. In summary, our work has revealed diverse dynamical systems underlying nongenetic heterogeneity of cells, which were previously underappreciated."
author2: "MeiLu McDermott"
inst2: "University of Southern California (Department of Biology)"
title2: "Characterizing Intermediate States of Epithelial-Mesenchymal Transition in Cancer through Single-Cell RNA Sequencing and Mathematical Modeling"
additional_authors2: ""
abstract2: "The epithelial-mesenchymal transition (EMT) is a primary biological mechanism of cancer metastasis, involving cells transforming from an adhesive epithelial phenotype to a migratory mesenchymal phenotype. Recent research has identified intermediate EMT states, characterized by hybrid phenotypes experimentally shown to be metastatic. This comparative study investigates these hybrid EMT cells across multiple cancers using single-cell RNA sequencing data. We identified genes upregulated in multiple intermediate EMT states across cancers, particularly those related to β-catenin regulation. Additionally, we developed a mathematical model using ordinary differential equations (ODEs) to describe EMT rates and fitted the model to scRNAseq data. Incorporating data from multiple cancer types, our ODE model provides a discovery tool for identifying genes associated with stabilizing the existence of metastatic, hybrid EMT cells."
author3: "Ken J. Oestreich"
inst3: "The Ohio State University School of Medicine (Microbial Infection and Immunity)"
title3: "Regulation of T helper cell programming by the transcription factor Aiolos"
additional_authors3: ""
abstract3: "CD4+ cytotoxic T lymphocytes, or CD4-CTLs, comprise an effector subset capable of performing cytotoxic functions normally associated with CD8+ T and Natural Killer cells. CD4-CTLs play critical roles in many immunological contexts, including protective anti-viral responses to influenza infection. Despite their well-documented importance to healthy immune responses, the regulatory mechanisms that underlie their formation remain unclear. We have identified the Ikaros transcription factor Aiolos as a novel repressor of cytoxic programming in CD4 T cells. We demonstrate that Aiolos deficiency results in increased CD4-CTL responses in the lungs of influenza-infected mice, as assessed by elevated expression of Granzyme B and Perforin, as well as the CTL marker NKG2A/C/E. We further find that Aiolos-deficient CD4-CTLs exhibit increased expression of transcription factors associated with cytotoxic programming, including Eomes and Blimp-1. Mechanistically, we demonstrate that Aiolos-deficient cells have a heightened sensitivity to IL-2/STAT5 signaling due to enhanced expression of the IL-2 cytokine receptor and that this translates into increased STAT5 association at regulatory regions of hallmark CD4-CTL genes. Intriguingly, the STAT5 motif partially overlaps with that of the core Aiolos DNA binding motif, suggesting that Aiolos may function to broadly antagonize STAT5 activity throughout the genome. Collectively, this work establishes Aiolos as a novel repressor of CD4-CTL differentiation and highlights its potential as a therapeutic target for enhancing anti-viral immune responses."
author4: "Rachel A. Gottschalk"
inst4: "University of Pittsburgh (Department of Immunology)"
title4: "Modeling cytokine-induced signaling features and sensitivity to network variation"
additional_authors4: "Neha Cheemalavagu, Department of Immunology, University of Pittsburgh School of Medicine; Laura F. Strube, Department of Immunology, University of Pittsburgh School of Medicine; Karsen E. Shoger, Department of Immunology, University of Pittsburgh School of Medicine; Yuqi M. Cao, Department of Immunology, University of Pittsburgh School of Medicine; Brandon A. Michalides, Department of Immunology, University of Pittsburgh School of Medicine; Samuel A. Botta, Department of Immunology, University of Pittsburgh School of Medicine; James R. Faeder, Department of Computational and Systems Biology, University of Pittsburgh School of Medicine"
abstract4: "Cells choose environment-specific functions by integrating stimuli through biochemical signaling pathways. Predicting functional outcomes of signaling is complicated by the complexity of network interactions and the diversity of signals which converge on a relatively small number of intracellular components. For example, over 50 cytokines and growth factors are distinguished by the JAK/STAT signaling pathways, comprised of 4 JAKs (Janus kinases) and 7 STATs (signal transducers and activators of transcription), to produce stimulus-specific cellular functions. In many cases, opposing phenotypes (pro- vs. anti-inflammatory) depend on the same STAT proteins to induce distinct patterns of gene expression. Predicting the relationship between signaling conditions and STAT phosphorylation profiles and then linking them to downstream gene expression remains an unaddressed challenge. 

We have developed a mechanistic-to-machine learning computational workflow that links STAT phosphorylation trajectories to global gene expression patterns via an ODE-simulated, rule-based model and machine learning. Our model is parameterized with STAT phosphorylation data from IL-6 and IL-10 stimulated macrophages. Machine learning is used to link these profiles to transcriptomic data under the same stimulation conditions. Parameter analysis of our mechanistic model identified JAK2 as having STAT-specific impacts on dynamic signaling features. Using the full computational workflow, we predicted and validated the impact of selective JAK2 inhibition on downstream gene expression and identified clusters of dynamically regulated genes that were sensitive and insensitive to JAK2 alterations. Thus, this work is an important step towards the use of multi-level prediction models to link stimuli to gene expression and to identify the effect of network perturbations. We are currently exploring sensitivity analysis approaches to derive biological insight from quantitative parameter relationships, with an interest in predicting parameters and parameter ratios that are highly sensitive to variation. Our objective is to enhance our understating of how altered expression of signaling network components impacts cellular responsiveness to cytokines and JAK inhibition in physiologically and clinically relevant contexts, such as cancer and human genetic variation."
---
