---
layout: group
poster: "ONCO-12"
day: "Monday"
permalink: /ONCO/PS01-ONCO-12.html
code: "PS01"
subgroup: "ONCO"
author: "Nicholas Harbour"
inst: "The University of Nottingham"
title: "Mathematical modelling of interacting sub-populations in glioblastoma"
abstract: "One of the major challenges in successfully treating glioblastoma (GBM) is the significant heterogeneity in cellular composition observed within and between patients. Recent single cell transcriptomics suggests there can be as many as eighteen distinct cell types in a single tumour [1]. Furthermore, advances in cellular deconvolution techniques, such as CIBERSORTx, allow us to accurately determine the cellular composition of imaged localised biopsies from bulk RNA-Seq [2]. Understanding this heterogeneity and how the complex interactions between cellular populations impacts the progression of GBM may lead to novel treatments which exploit the unique cellular composition within individual tumours. We group these eighteen cell types into sub-populations, e.g., glioma, immune, astrocyte, then attempt to learn the dynamics of these sub-populations by considering various interacting ODE/PDE models. Typically, a GBM patient will have biopsies taken at most twice, as well as only a handful of MRI scans. Therefore, the number of temporal data points to fit any model to are very limited. Thus, we apply trajectory inference methods, such as Monocle, to biopsy data, which allows us to order samples via pseudotime, an arbitrary unit of progress akin to real time [3]. We illustrate our modelling approach with a simplified two species Lotka-Volterra style competition model. 



[1] O. Al-Dalahmah, et al., Re-convolving the compositional landscape of primary and recurrent glioblastoma using single nucleus RNA sequencing. bioRxiv (2021) https://doi.org/10.1101/2021.07.06.451295 

[2] C. B. Steen, C. L. Liu, A. A. Alizadeh, A. M. Newman, Profiling Cell Type Abundance and Expression in Bulk Tissues with CIBERSORTx. Methods Mol. Biol. 2117, 135–157 (2020). 

[3] C. Trapnell, et al., The dynamics and regulators of cell fate decisions are revealed by pseudotemporal ordering of single cells. Nat. Biotechnol. 32, 381–386 (2014)."
additional_authors: "Markus Owen (University of Nottingham); Matthew Hubbard (University of Nottingham); Michael Chappell (University of Nottingham); Kristin Swanson (Mathematical neuro-oncology lab Mayo Arizona); Lee Curtin (Mathematical neuro-oncology lab Mayo Arizona)"
---
