---
layout: group
author: smb2023
day: "Tuesday"
subgroup: "ONCO"
subgroup-long: "ONCO-1"
title: "Dynamics of cellular heterogeneity: consequences of diverse regulatory mechanisms"
organizers: "Mohit Kumar Jolly, Paras Jain"
description: "The advent of new experimental technologies and computational tools over past two decades have helped us to capture heterogeneity seen at the single-cell level. Further, the role of this heterogeneity in enabling a higher fitness of cancer cell population has been demonstrated. Moving ahead, a key focus is to understand the regulatory and stochastic origins of cellular heterogeneity so as to modulate it to achieve desired outcomes, such as better therapeutic outcomes. Here, we bring together a diverse set of experts who employ experimental and mathematical tools to understand the origins and consequences of heterogeneity witnessed in a cell population. Further, the symposium will also fuel the need to integrate information at multiple levels of intra-cellular/extra-cellular regulations to improve our understanding of functional responses at the single-cell/population levels."
room: "Ohio Staters Traditions Room (#2120)"
code: "MS03"
code2: "MS04"
author1: "Amy Brock"
inst1: "University of Texas at Austin (Biomedical Engineering)"
title1: "Heritability and plasticity of therapeutic resistance mechanisms within heterogeneous cancer cell populations"
additional_authors1: ""
abstract1: "Individual cancer cells within a tumor cell population display distinct responses to chemotherapeutic agents. We have developed a novel genetic tracking technology, ClonMapper to elucidate the pre-existent and de novo cell states that arise from chemotherapeutic intervention. By tracking longitudinal clonal dynamics and cell state changes, we elucidate the contributions of heterogeneity to survival and re-growth of cancer cells following specific selective pressures. Here we will examine the distinct survivorship trajectories that characterize breast cancer cells treated with chemotherapy. Subpopulations differ significantly in growth dynamics and the interactions among heterogeneous subpopulations impact the population composition of surviving cells. Models that include subpopulation interactions may improve the ability to predict the composition and sensitivity of cancer cells under varying therapeutic pressures."
author2: "Morgan Craig"
inst2: "Sainte-Justine University Hospital Research Centre / Université de Montréal (Immune Disorders and Cancer / Mathematics and Statistics)"
title2: "Impact of cellular and spatial heterogeneity on immunotherapies to treat glioblastoma"
additional_authors2: "Anudeep Surendran, Sainte-Justine University Hospital Research Centre / Université de Montréal; Adrianne Jenner, Queensland University of Technology"
abstract2: "Glioblastoma is a rare but deadly central nervous system and brain cancer. In most patients, current standard-of-care, which includes maximal safe surgical resection, radiotherapy, and chemotherapy, fails due to recurrences, translating to a median survival of just 15 months. There is therefore high interest in developing improved approaches to treat glioblastoma, including immunotherapies (e.g., immune checkpoint blockade, oncolytic viruses etc.). Unfortunately, clinical trials of immunotherapies to treat glioblastoma have thus far failed to show significant benefits to patients. In this talk, I will discuss the role of spatial and cellular heterogeneity on treatment success through the integration of agent-based modelling with clinical samples from patients. Our results suggest avenues of continued drug development to provide improved patient outcomes."
author3: "Yogesh Goyal"
inst3: "Northwestern University (Cell and Developmental Biology)"
title3: "Tracing origin and consequences of rare cell plasticity in cancer drug resistance"
additional_authors3: ""
abstract3: "Single cell variations within a genetically homogeneous population of cells can lead to significant differences in cell fate in response to external stimuli. This is particularly relevant in cancer cells, where a small population of cells can evade therapies to develop resistance. In this talk, I will present ongoing work on tracing the origins, nature, and manifestations of single cell variations in response to a variety of cytotoxic chemotherapies and targeted therapies in various cancer models. By combining clonal barcoding-based and imaging-based lineage tracing frameworks with computational analysis, I will discuss the commonalities and differences in cell fate outcomes across cancers and therapies. Our experimental and computational designs will provide a foundation for controlling single-cell variabilities in cancer and other biological contexts, such as stem cell reprogramming."
author4: "Geena Ildefonso"
inst4: "University of Southern California (Alfred E. Mann Department of Biomedical Engineering, University of Southern California, Los Angeles, California, USA)"
title4: "A data-driven Boolean model explains memory subsets and evolution in CD8+ T cell exhaustion"
additional_authors4: "Stacey D. Finley (1,2,3)
1 Alfred E. Mann Department of Biomedical Engineering, University of Southern California, Los Angeles, California, USA
2 Department of Quantitative and Computational Biology, University of Southern California, Los Angeles, California, USA
3 Mork Family Department of Chemical Engineering and Materials Science, University of Southern California, Los Angeles, California, USA"
abstract4: "T cells play a key role in a variety of immune responses, including infection and cancer. Upon stimulation, naïve CD8+ T cells proliferate and differentiate into a variety of memory and effector cell types; however, failure to clear antigens causes prolonged stimulation of CD8+ T cells, ultimately leading to T cell exhaustion (TCE). The functional and phenotypic changes that occur during CD8+ T cell differentiation are well characterized, but the underlying gene expression state changes are not completely understood. Here, we utilize a previously published data-driven Boolean model of gene regulatory interactions shown to mediate TCE. Our network analysis and modeling reveal the final gene expression states that correspond to TCE, along with the sequence of gene expression patterns that give rise to those final states. With a model that predicts the changes in gene expression that lead to TCE, we could evaluate strategies to inhibit the exhausted state. Overall, we demonstrate that a common pathway model of CD8+ T cell gene regulatory interactions can provide insights into the transcriptional changes underlying the evolution of cell states in TCE."
---
