---
layout: group
author: smb2023
day: "Thursday"
subgroup: "MFBM"
subgroup-long: "MFBM-2"
title: "Data-driven multiscale modeling of cancer"
organizers: "Heber Rocha, John Metzcar,
Paul Macklin"
description: "The integration of data, such as multiplexed -omics data, histopathology slides, or cell viability and motility assays, into multiscale simulations is a complex and challenging problem. The available data is often high-dimensional, noisy, sparse, and variable in both time and space. On the modeling side, there are many parameters to consider, and numerous possible emergent outcomes may arise. The first step in this process is to map the data to model parameters. This is often a difficult task due to the complexity of the data and the large number of parameters that must be considered. Once the data has been mapped to the model parameters, it is crucial to confirm that the model can reproduce the observations with appropriate fidelity. This step is critical to ensuring that the model is accurate and reliable. In this mini-symposium, we explore new and emerging techniques to address these challenges. We examine the integration of qualitative and quantitative data at each level of the modeling process. Additionally, we discuss the impacts of parameter uncertainty in the quality of model predictions. Overall, this mini-symposium focuses on the challenges of integrating data into multiscale simulations and the techniques and strategies that can be used to overcome these challenges."
room: "Suzanne M. Scharer Room (#3146)"
code: "MS06"
author1: "Alexander Browning"
inst1: "University of Oxford (Mathematical Institute)"
title1: "Drawing biological insight from non-identifiabile models of tumour growth using simple surrogates"
additional_authors1: "Matthew Simpson (Queensland University of Technology)"
abstract1: "Models are now routine in the interpretation of biological data, however are often limited by parameter non-identifiabilities. Indeed, simple goodness-of-fit metrics including the likelihood and residual error give only limited information about where a model does and doesn’t fit. In the talk, we demonstrate a new framework for the study non-identifiability of complex models of tumour growth using simple surrogate models, that lie in between a model of interest and the data. For example, the traditional one-dimensional line of constant goodness-of-fit, studied in traditional likelihood-based identifiability analysis, might lie at the intersection of two higher dimensional surfaces, each representing features in the data (in this case, the maximum size and initial growth rate of the tumour). One can move along this intersection in parameter space and achieve only a minimal change to the model predictions; hence, parameter non-identifiability. Overall, we demonstrate a novel technique for gaining insight from the complex biological models that are often essential for the elucidation of important biological processes."
author2: "Jeanette Johnson"
inst2: "Johns Hopkins University (Immunology)"
title2: "Integrating Omics Data and Agent-Based Models for Comprehensive Digital Biology"
additional_authors2: "Max Booth, Johns Hopkins University; Heber Rocha, Indiana University; Atul Deshpande, Johns Hopkins University; Ines Godet, Memorial Sloan Kettering Cancer Center; Jacob T. Mitchell, Johns Hopkins University; Ines Godet, Johns Hopkins University; Daniele Gilkes, Johns Hopkins University; Elizabeth Jaffee, Johns Hopkins University; Lei Zheng, Johns Hopkins University; Jacquelyn Zimmerman, Johns Hopkins University; Genevieve Stein-O’Brien, Johns Hopkins University; Elana J Fertig, Johns Hopkins University; Paul Macklin, Indiana University"
abstract2: "Agent-based modeling for biological systems currently suffers from limited ability to systematically integrate experimental data into model parameters. To facilitate the wider use of ABMs in these contexts, we sought ways to address this issue. Here we present two models built in the PhysiCell agent based modeling framework using results from high-throughput analyses. We show a method for generation of model agents directly from standard 10x Visium spatial transcriptomics data into an initial state of the ABM, preserving spatial relationships between cells when translating data to model. In future work we hope to build a general purpose “digital tissue” pipeline, capable of constructing spatially-resolved ABMs in 2d and 3d. We then show a model built upon more qualitative observations from a pancreatic cancer single-cell atlas and a separate set of clinical trial results from a matched biological context. The model simulates hypotheses of tumor progression among a group of epithelial and immune cells under three therapy conditions, exploring the tumor-immune logic of the system and attempting to understand what could have led to the results of the clinical trial. These integrations are proof-of-concept for more general integration of omics with agent-based models, and will empower life science investigators to make use of ABMs, which are ultimately very powerful hypothesis building and visualization tools, in their research."
author3: "Adam MacLean"
inst3: "University of Southern California (Department of Quantitative and Computational Biology)"
title3: "Learning gene regulatory networks that control cell state transitions from multi-modal single-cell genomics"
additional_authors3: ""
abstract3: "Single-cell genomics offer unprecedented resolution with which to study cell fate decision-making in cancer. We present new tools to infer gene regulatory networks (GRNs) controling cell fate decisions and model their multiscale dynamics. We introduce popInfer, single-cell multi-modal GRN inference via regularized regression, and demonstrate its potential for network discovery Through application to hematopoiesis, we discover new gene interactions regulating early fate decisions during stem cell differentiation that are profoundly affected by diet and age."
author4: "Matthew Simpson"
inst4: "Queensland University of Technology (School of Mathematical Sciences)"
title4: "A stochastic mathematical model of 4D tumour spheroids with real-time fluorescent cell cycle labelling"
additional_authors4: ""
abstract4: "In vitro tumour spheroids have been used to study avascular tumour growth and drug design for over 50 years. Tumour spheroids exhibit heterogeneity within the growing population that is thought to be related to spatial and temporal differences in nutrient availability. The recent development of real-time fluorescent cell cycle imaging allows us to identify the position and cell cycle status of individual cells within the growing spheroid, giving rise to the notion of a four-dimensional (4D) tumour spheroid. We develop the first stochastic individual-based model (IBM) of a 4D tumour spheroid and show that IBM simulation data compares well with new experimental data using a primary human melanoma cell line. The IBM provides quantitative information about nutrient availability within the spheroid, which is important because it is difficult to measure these data experimentally."
---
